sandstone_jupyterhub_login/handlers.py

```diff
@@ -32,3 +32,15 @@ def get(self):
         else:
             self.set_status(403)
             self.redirect(self.get_login_url())
+
+
+class JupyterHubLogoutHandler(BaseHandler):
+    def get(self):
+        self.xsrf_token
```

Comment:
This line sets the XSRF token if it does not already exist, otherwise it simply references its value. Either way, it is unnecessary in a logout handler.